src/Hl7.Fhir.Core/Hl7.Fhir.Core.csproj

```diff
@@ -17,9 +18,16 @@
     <RootNamespace>Hl7.Fhir</RootNamespace>
   </PropertyGroup>
 
+   <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net45|AnyCPU'">
+     <DefineConstants>TRACE;USE_CODE_GEN</DefineConstants>
+   </PropertyGroup>
+
   <ItemGroup>
```

Comment:
I am not sure of this is needed: Compile, EmbeddedResource.. I even think that the whole `ItemGroup` can be removed

Reply:
removed